test/torch_np/test_reductions.py

```diff
@@ -302,6 +316,10 @@ def test_sum_where(self):
     np.count_nonzero,
 }
 
+restricts_dtype_casts = {np.var, np.std}
+
+fails_empty_tuple = {np.argmin, np.argmax}
```

Comment:
These we can fix later right?

Reply:
`fails...` is not the right word probably. We allow it while numpy does not:

```
In [1]: import numpy as np

In [2]: np.argmin(np.ones(3), axis=())
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
File ~/mambaforge/envs/scipy-dev/lib/python3.10/site-packages/numpy/core/fromnumeric.py:59, in _wrapfunc(obj, method, *args, **kwds)
     58 try:
---> 59     return bound(*args, **kwds)
     60 except TypeError:
     61     # A TypeError occurs if the object does have such a method in its
     62     # class, but its signature is not identical to that of NumPy's. This
   (...)
     66     # Call _wrapit from within the except clause to ensure a potential
     67     # exception has a traceback chain.

TypeError: 'tuple' object cannot be interpreted as an integer

During handling of the above exception, another exception occurred:

TypeError                                 Traceback (most recent call last)
Cell In[2], line 1
----> 1 np.argmin(np.ones(3), axis=())

<snip>

File ~/mambaforge/envs/scipy-dev/lib/python3.10/site-packages/numpy/core/fromnumeric.py:45, in _wrapit(obj, method, *args, **kwds)
     43 except AttributeError:
     44     wrap = None
---> 45 result = getattr(asarray(obj), method)(*args, **kwds)
     46 if wrap:
     47     if not isinstance(result, mu.ndarray):

TypeError: 'tuple' object cannot be interpreted as an integer

```